.circleci/config.yml

```diff
@@ -3,19 +3,19 @@ jobs:
   build:
     working_directory: ~/app
     docker:
-      - image: circleci/node:8.9.4
+      - image: circleci/node:8.10.0
```

Comment:
eslint v5 [expects](https://github.com/eslint/eslint/blob/master/package.json#L126) versions `^6.14.0 || ^8.10.0 || >=9.10.0`. All consuming repos we'll likely need to upgrade if they haven't already.